dev/docker/ranger/Dockerfile

```diff
@@ -59,20 +25,18 @@ RUN ARCH=$(uname -m) && \
     ln -s /usr/lib/jvm/java-11-openjdk-amd64 ${JAVA_HOME}; \
     fi
 
-COPY --from=compile-ranger /root/apache-ranger/target/ranger-${RANGER_VERSION}-admin.tar.gz /opt
-RUN cd /opt && \
-    tar zxvf ranger-${RANGER_VERSION}-admin.tar.gz && \
-    ln -s ranger-${RANGER_VERSION}-admin ranger-admin
+# install ranger
+ENV RANGER_HOME=/opt/ranger-admin
+RUN mkdir -p ${RANGER_HOME}
+ADD packages/ranger-${RANGER_VERSION}-admin.tar.gz /opt/
+RUN ln -s /opt/ranger-${RANGER_VERSION}-admin/* ${RANGER_HOME}
```

Comment:
I think we didn't need to mkdir `${RANGER_HOME}`, We can directly create safe link
```
ADD packages/ranger-${RANGER_VERSION}-admin.tar.gz /opt/
RUN ln -s /opt/ranger-${RANGER_VERSION}-admin ${RANGER_HOME}
```

Reply:
If i remove mkdir, I got:

```
 > [ 9/12] RUN ln -s /opt/ranger-2.4.0-admin/* /opt/ranger-admin:
2.597 ln: target '/opt/ranger-admin' is not a directory
```

Reply:
You need to change `ln` command params.
RUN ln -s `/opt/ranger-${RANGER_VERSION}-admin` ${RANGER_HOME}

Reply:
okay, fixed.